deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/worker/DeployTagWorker.java

```diff
@@ -94,41 +106,12 @@ private void processEachEnvironConstraint(DeployConstraintBean bean) throws Exce
         }
 
         // 2. add host tags from missing in db ( query for CMDB for the missing host tags )
+        // No need at all since we already updated the tags via TAS
+        // For now we emit metrics to monitor
         if (!missings.isEmpty()) {
-            List<UpdateStatement> statements = new ArrayList<>();
```

Comment:
please keep the logic until we have metrics showing that it's not needed.

Reply:
sounds good